src/components/TabBar/TabBar.jsx

```diff
@@ -62,7 +63,14 @@ const TabBar = ({ tabs, info, match }) => (
           disabled={tab.disabled}
           hidden={tab.hidden && tab.hidden(match)}
         >
-          {tab.name}
+          <div data-tip={tab.tooltip && true} data-for={`tooltip_${tab.key}`}>
```

Comment:
what's the purpose of the && true?

Reply:
@howardchung I saw something similar for the tooltips in TableHeaderColumn.jsx . I think the idea is to have a simple value in the attribute that you can use for CSS selecting, like: &[data-tip="true"] {...}

Reply:
@howardchung  ReactTooltip finds the tooltips with the data-tip attribute and it cannot be undefined.

Reply:
actually nevermind. it has to be ```data-tip={tab.tooltip}``` or else the mouse cursor will change on tabs where there is no tooltip. see the heroes page for example

Reply:
@Argn0 is there value in me keeping the value as `{tab.tooltip && true}` so that rules like `[data-tip=true]` can still work? Or would the simpler code `data-tip={tab.tooltip}` be preferred.

Reply:
I'd just use ```data-tip={tab.tooltip}``` because the [data-tip=true] rule was removed.

Reply:
I've removed the `&& true`.